molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java

```diff
@@ -2010,6 +2014,33 @@ public void update()
 	}
 
 	@Test
```

Comment:
You can set a test to expect an exception:

`@Test(expectedExceptions = { MolgenisDataException.class }, expectedExceptionsMessageRegExp = "The entity you are trying to update [id999] does not exist.")`